problems/variables.js

```diff
@@ -7,6 +7,12 @@
 //  * For example, if you were born in 1988, then in 2026 you'll be either 37 or 38, depending on what month it is in 2026.
 //  * Log them to the screen like so: "I will be either `ageMin` or `ageMax` in `futureYear`", substituting the values.
 
+// const birthYear = 1977;
+// let futureYear = 2030;
+// let futureAge = futureYear - birthYear;
```

Comment:
There should be two options for your age!

Reply:
Ohh I didn't read it properly!!